File: algorithms/easy/find_champion_i.dart
class Solution {
  int findChampion(List<List<int>> grid) {
    int ct = 0;
    int team = 0;
    for (int i = 0; i < grid.length; i++) {
      int len = grid[i].where((e) => e == 1).length;
      if (ct < len) {
        ct = len;
        team = i;
      }
    }
    return team;
  }
}

File: algorithms/easy/find_champion_i.rs
impl Solution {
    pub fn find_champion(grid: Vec<Vec<i32>>) -> i32 {
        let mut team = 0;
        let mut ct = 0;
        for (i, row) in grid.into_iter().enumerate() {
            let len: i32 = row.into_iter().reduce(|acc, e| acc + e).unwrap();
            if (ct < len) {
                team = i;
                ct = len;
            }
        }
        team as i32
    }
}
